### PR TITLE
Clean up login pattern text

### PR DESCRIPTION
**Pull request by yih-wang**

### CODE REVIEW

**Comment by yih-wang** [comment]
@mcarrano Updated all the mentioned text. Plz take a look.

File: pattern-library/application-framework/login-page/design/design.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ In most cases, client-side validations are performed as inline validations for e
 1. **When to Show**: When the user clicks the action button (Log In/Next), the errors must be corrected before the form can be submitted, and an error message will be displayed below any empty field.
 2. **When to Hide**: When the user fills the empty field, the error message will disappear when the field loses focus.
 3. **Recommended Message**:
-  - When the username field is empty, we recommend to show “Enter your username”. “Username” can be replaced by “email” or other words if username is restricted to specific types of words.
+  - When the username field is empty, we recommend to show “Enter your email or phone or username”. Email/phone/username can be restricted to specific types according to the target application.
```
[the remark this answers is not in the document]
Comment (yih-wang) [reply pattern-library/application-framework/login-page/design/design.md]: In the latest mockup, the placeholder of username field is "Email or phone or username". And when the username field is empty, the error message is displayed as followed image.
message-1
So I modified the text to keep consistency. Do you think we should remove "phone" from these places?

File: pattern-library/application-framework/login-page/design/design.md [hunk 2]
```diff
@@ -20,8 +20,8 @@ Try to limit the total number of fields to less than five. Also, you may use pro
 
 ### Social Login
 ![Image of social login](img/login-page-4.png)
-1. **Social Media Buttons**: When the number of buttons is four or less, display all buttons in one column. Otherwise, use a two-column layout.
-2. **More Button**: When there are more than 8 buttons, just show first 7 of them. User can click the More Button to expand the list and view all the buttons as followed screen.
+1. **Social Media Buttons**: When the number of buttons is four or less, display all buttons in one column. Otherwise, use a two-column layout. All the buttons should be listed in the order of their mostly likely usage, that is, the most frequently used items should show at or near the top of the list.
+2. **More Link**: When there are more than 8 buttons, a more link will show below the grid on 8 buttons. User can click the link to expand the button grid and the link will change to "less".
```
[the remark this answers is not in the document]
Comment (yih-wang) [reply pattern-library/application-framework/login-page/design/design.md]: Fixed.

File: pattern-library/application-framework/multi-factor-login/design/design.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 ![Image of identity first](img/login-page-2.png)
 1. **Logotype**: The product logo is placed at the top.
 2. **Description** (optional): Description of the website can be placed here.
-3. **Title**: The title is located at the top of the login card.
+3. **Title**: The title is located at the top of the login card. "Your Account" can be replaced with any word that makes sense for the application.
```
[the remark this answers is not in the document]
Comment (yih-wang) [reply pattern-library/application-framework/multi-factor-login/design/design.md]: Fixed.

File: pattern-library/application-framework/login-page/design/design.md [hunk 4]
```diff
@@ -71,10 +71,8 @@ If the user input a wrong account name which do not exist or if the user input a
 
 1. Directly tell the user that the account doesn’t exist or the password is wrong. Although this approach is useful and clear for user, it's not recommended if protecting identity of valid users is important.
 ![Image of account or password errors](img/server-1.png)
- - When username is wrong (doesn’t exist), we recommend to show:
-  - "The account doesn’t exist. Try again."
-  - “The account doesn’t exist. Try again or Sign up for PatternFly.” for the products supporting signing up. The “Sign up for PatternFly” should be a link to help users creating a new account more easier. “PatternFly” can be replaced with other product names.
- - When password is wrong, we recommend to show “Incorrect password, please try again.” If the user inputs an old password (they may have changed the password recently), we can show an error message alerting them that the password has changed instead of stating “the password is wrong” only.
+ - When username is wrong (doesn’t exist), we recommend to show “The account does not exist. Try again or sign up”. The “sign up” should be a link to help users creating a new account much easier.
```
[the remark this answers is not in the document]
Comment (yih-wang) [reply pattern-library/application-framework/login-page/design/design.md]: Fixed.

File: pattern-library/application-framework/login-page/design/design.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mcarrano) [reply pattern-library/application-framework/login-page/design/design.md]: I guess it's OK to leave this way since it's already in the approved mockups.

**Comment by mcarrano** [comment]
Looks good @yih-wang . Thanks for making these updates. I will go ahead and merge.

**Pull request merged**